xbmc/pvr/PVRDatabase.cpp

```diff
@@ -229,6 +229,12 @@ void CPVRDatabase::UpdateTables(int iVersion)
 
   if (iVersion < 37)
     m_pDS->exec("ALTER TABLE channelgroups ADD iLastOpened integer");
+
+  bool bisMySQL = StringUtils::EqualsNoCase(
+      CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_databaseTV.type, "mysql");
+
+  if (iVersion < 38 && bisMySQL)
```

Comment:
No need to always obtain the value of `bIsMySQL`.

=> `if (iVersion < 38 && StringUtils::EqualsNoCase(
      CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()->m_databaseTV.type, "mysql"))`

Reply:
done